luajit.krun

```diff
@@ -32,6 +26,23 @@ ITERATIONS_ALL_VMS = 1500
 ITERATIONS_NO_JIT = 100
 INSTRUMENT = False
 
+def make_env(vmdir, extra=None):
+    basedir = os.path.join(DIR, "builds", vmdir)
+    ret = {
+        'LUA_PATH' : VM_ENV + '"{0}/?/init.lua;{0}/?.lua;{0}/?/?.lua;"'.format(basedir)
+    }
+    if extra != None:
+        ret.update(extra)
+    return ret
+
+def makevmdef(dirname, instrument=False, env_extra=None, n_iterations=None):
+    ret = {
```

Comment:
Should just be `return { ... }?` and drop the `ret =`?

Reply:
fixed